aspnetcore/blazor/components/render-modes.md

```diff
@@ -70,137 +72,122 @@ Example 1: The following `Program` file API adds services and configuration for
 
 ```csharp
 builder.Services.AddRazorComponents()
-    .AddServerComponents();
+    .AddInteractiveServerComponents();
 ```
 
 ```csharp
 app.MapRazorComponents<App>()
-    .AddServerRenderMode();
+    .AddInteractiveServerRenderMode();
 ```
 
 Example 2: The following `Program` file API adds services and configuration for enabling the WebAssembly render mode:
 
 ```csharp
 builder.Services.AddRazorComponents()
-    .AddWebAssemblyComponents();
+    .AddInteractiveWebAssemblyComponents();
 ```
 
 ```csharp
 app.MapRazorComponents<App>()
-    .AddWebAssemblyRenderMode();
+    .AddInteractiveWebAssemblyRenderMode();
 ```
 
-Example 3: The following `Program` file API adds services and configuration for enabling the Server, WebAssembly, and Auto render modes:
+Example 3: The following `Program` file API adds services and configuration for enabling the interactive Server, WebAssembly, and automatic (Auto) render modes:
 
 ```csharp
 builder.Services.AddRazorComponents()
-    .AddServerComponents()
-    .AddWebAssemblyComponents();
+    .AddInteractiveServerComponents()
+    .AddInteractiveWebAssemblyComponents();
 ```
 
 ```csharp
 app.MapRazorComponents<App>()
-    .AddServerRenderMode()
-    .AddWebAssemblyRenderMode();
+    .AddInteractiveServerRenderMode()
+    .AddInteractiveWebAssemblyRenderMode();
 ```
 
 Blazor uses the Blazor WebAssembly hosting model to download and execute components that use the WebAssembly render mode. A separate client project is required to set up Blazor WebAssembly hosting for these components. The client project contains the startup code for the Blazor WebAssembly host and sets up the .NET runtime for running in a browser. The Blazor Web App template adds this client project for you when you select the option to enable WebAssembly interactivity. Any components using the WebAssembly render mode should be built from the client project, so they get included in the downloaded app bundle.
 
 ## Apply a render mode to a component instance
 
-To apply a render mode to a component instance use the [`@rendermode` Razor directive attribute](xref:mvc/views/razor#attribute) where the component is used.
+To apply a render mode to a component instance use the [`@rendermode` Razor directive](xref:mvc/views/razor#rendermode) where the component is used.
 
 In the following example, the Server render mode is applied to the `Dialog` component instance:
 
 ```razor
-<Dialog @rendermode="@RenderMode.Server" />
+<Dialog @rendermode="RenderMode.InteractiveServer" />
 ```
 
 > [!NOTE]
-> During .NET 8 *Release Candidate 1*, use the following values:
+> The preceding syntax will be simplified at the release of .NET 8 in November to:
```

Comment:
We could just tell people here that they can add the `using state` for `RenderMode` in their *_Imports.razor* to get the shorter syntax. That's what we plan to add in the Blazor Web App template.